### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by hydaline-code**
Hi 👋 in this part of the project, the following has been added :

✅Created a new directory called components.
✅Inside components, created a new Calculator.js file.
✅Inside components, created a new Calculator.css file.
✅In Calculator.js, A React component that matches this design has been implemented.

Thanks 🙏

### CODE REVIEW

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @hydaline-code  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ React is set properly
✅ Good design👍 
✅ Professional documentation👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ['1', '2', '3', '+'],
+    ['0', '.', '='],
+  ];
+  return (
+    <div className="calculator">
+      <div className="display">{displayValue}</div>
+      <div className="buttons">
+        {buttonRows.map((row, rowIndex) => (
+          <div className="row" key={uuidv4()}>
+            {row.map((button, columnIndex) => {
```
Comment (ghost) [inline src/components/Calculator.js]: Great job so far! 👏 However, for the calculator component, we need to have at least one child component. 🧒🔢 Could you please implement that? Thanks! 😊

**Review by frank1738 — CHANGES_REQUESTED** [review]
Hi @hydaline-code  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!




## Highlights

✅ React is set properly
✅ Good design👍 
✅ Professional documentation👍 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

N/A
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @hydaline-code, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- Gitflow is followed. ✔️
- Good job on implementing the changes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**